python/servo/testing_commands.py

```diff
@@ -30,7 +30,10 @@
 from update import updatecommandline
 import tidy
 
-here = os.path.split(__file__)[0]
+SCRIPT_PATH = os.path.split(__file__)[0]
+PROJECT_TOPLEVEL_PATH = os.path.abspath(os.path.join(SCRIPT_PATH, "..", ".."))
```

Comment:
I don't know how Windows works, so I usually prefer to use the build-in cross-platform path manipulation tools. What you have is probably fine.